.eleventy.js

```diff
@@ -1,61 +1,16 @@
-/*
-Copyright the eleventy-plugin-fluid copyright holders.
-
-See the AUTHORS.md file at the top-level directory of this distribution and at
-https://github.com/fluid-project/eleventy-plugin-fluid/raw/main/AUTHORS.md.
-
-Licensed under the New BSD license. You may not use this file except in compliance with this License.
-
-You may obtain a copy of the New BSD License at
-https://github.com/fluid-project/eleventy-plugin-fluid/raw/main/LICENSE.md.
-*/
```

Comment:
Was the removal of the copyright statement intentional?

Reply:
This is actually a new file. The Eleventy plugin's main file has been renamed to `index.js` and I added this file solely for the purposes of testing the UIO integration within this repository. As a test file which won't be used in projects which include this as a dependency, I didn't think to add a copyright statement but if you think it's necessary I can.

Reply:
Thanks for the clarification. We tend to add copyright statements to all of the JS files, so might as well.